files/zh-cn/glossary/abstraction/index.md

```diff
@@ -1,18 +1,20 @@
 ---
 title: 抽象编程（Abstraction）
 slug: Glossary/Abstraction
+l10n:
+  sourceCommit: acf33012514328a403c162f65a73e9aec4e766f3
```

Comment:
Haven't notice this, the value should be: `ada5fa5ef15eadd44b549ecf906423b4a2092f34`

Reply:
yeap…will open another pr